build.gradle

```diff
@@ -12,6 +13,7 @@ buildscript {
 
     repositories {
         google()
+        mavenCentral()
```

Comment:
my understanding is that jcenter is a supercet of mavenCentral do you need both?